go.mod

```diff
@@ -40,4 +40,7 @@ replace (
 	k8s.io/client-go => k8s.io/client-go v0.19.7
 	k8s.io/code-generator => k8s.io/code-generator v0.19.7
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
+	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20210112194111-24e8ffe5a3a5
```

Comment:
Why is this version so wild: `v0.99.1-0.20210112194111-24e8ffe5a3a5`?

Reply:
This version is automatically resolved/updated by `go get` (or ./hack/update-deps.sh) when we point to the given branch (release-0.20).

Reply:
It's likely because we don't tag that repo.